src/error.ts

```diff
@@ -1,5 +1,8 @@
 import { Status } from "./constants.ts";
 
+export const ERROR_TRANSACTION_FINALIZED =
```

Comment:
I'm not sure if I like this or not, but generally: if we want to move all error strings into one place, we should do it in a dedicated pull (there are some in other places as well).

Generally I might prefer to not do this though, so we can keep the error strings close to where the error originates. (They don't need to/ shouldn't be accessible for e.g. programmatically checking what the error is, since they are meant for human consumption and not part of the stable API.)

Reply:
No problem, I changed it back :)